### PR TITLE
upgrade to webpack5

### PR DESCRIPTION
**Pull request by nikhilmandlik**
### Resolves: 
resolves: #3899

### Author Checklist:
Check | Passed
-- | --
Changes address original issue? | Y
Unit tests included and/or updated with changes? | N/A
Command line build passes? | Y
Changes have been smoke-tested? | Y

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const webpackConfig = {
         maelstromTheme: './src/plugins/themes/maelstrom-theme.scss'
     },
     output: {
+        publicPath: '',
```
Comment (shefalijoshi) [inline webpack.config.js:30]: Is this needed?

**Review by shefalijoshi — COMMENTED** [review]
Looks good. One comment

File: webpack.config.js (hunk 1 above)
Comment (nikhilmandlik) [reply webpack.config.js:30]: yes without it `npm install` fails.
Error: Automatic publicPath is not supported in this browser

Comment (shefalijoshi) [reply webpack.config.js:30]: Actually, can you try to upgrade the mini-css-extract-plugin to 1.3.9 and remove the publicPath? I think that fixes the issue.

**Review by shefalijoshi — CHANGES_REQUESTED** [review]
Please see comment to remove the publicPath change.

**Review by shefalijoshi — CHANGES_REQUESTED** [review]
In webpack.config.js stats.warningsFilter should be replaced with:
`ignoreWarnings: [/asset size limit/g]`
See: https://webpack.js.org/configuration/#

**Comment by nikhilmandlik** [comment]
> In webpack.config.js stats.warningsFilter should be replaced with:
> `ignoreWarnings: [/asset size limit/g]`
> See: https://webpack.js.org/configuration/#

hmm, I checked old builds, looks like this is not working at all (eg: [here](https://app.circleci.com/pipelines/github/nasa/openmct/4451/workflows/30e3ecf1-22f3-4099-9b77-bb48a76c844e/jobs/13296?invite=true#step-104-62)), may be we can remove it and let it warn about asset size.

**Review by shefalijoshi — APPROVED** [review]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue) Y

**Pull request merged**